static/javascripts/components/survey_card.jsx

```diff
@@ -2,6 +2,7 @@
 /*
 * Survey. Filled with data to send (JSON)
 * survey_handler.py/def _survey_from_request(self):
+* models/question.py
 */
 var survey = [
```

Comment:
@wannabeCitizen I am creating a variable to send the survey, let me know what you think.